### PR TITLE
Feat cart management

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Cart Management
- On the product card,
    - Quantity of a particular product and increase or decrease the same
    - Remove the product from the cart
    - Add the product to the Wishlist
- price details card of the cart containing a button to checkout which will show the total price of the products with its quantity.

Added some manual values instead of variables to just check how does it go. I would change later.

screenshot (5)

### CODE REVIEW

File: css/cart-page.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+.place-order {
+    padding: 5px;
+    width: 100%;
+    text-transform: capitalize;
+    border: transparent;
+}
+
+.place-order:hover {
+    background-color: #f13737;
```
Comment (Hamdrive) [inline css/cart-page.css]: Consider using CSS variables

File: css/helper.css [hunk 2]
```diff
@@ -12,3 +12,8 @@
     gap: 2rem;
 }
 
+.grid-75-25 {
+    grid-template-columns: 60% 1fr;
```
Comment (Hamdrive) [inline css/helper.css:17]: You can instead define a new class, given that the name and property being defined dont match
like `grid-60-40`

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Great work done, site UI looks super amazing!
Have suggested a change for helper class, would like to see it rectified

File: css/cart-page.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+.free {
+    margin-left: auto;
+    flex-basis: 19%;
+}
+
+.free span {
+    color: var(--cta-color);
+}
```
Comment (saadahmadhere) [inline css/cart-page.css]: consider giving EOF line here.

**Review by saadahmadhere — APPROVED** [review]
The UI looks great. Code is neat and nicely written. Appropriate comments have been made earlier. Good to merge.

File: css/helper.css (hunk 2 above)
Comment (hinanshisuthar) [reply css/helper.css:17]: yes it is indeed a new class but as I was experimenting, at the end  forgot to change it to 60-40

File: css/cart-page.css (hunk 1 above)
Comment (hinanshisuthar) [reply css/cart-page.css]: would make the change

File: css/cart-page.css (hunk 3 above)
Comment (hinanshisuthar) [reply css/cart-page.css]: Beautify removes it. Would add it.

**Review by Hamdrive — APPROVED** [review]
Great work done, changes implemented looks good!
ready to merge!

**Pull request merged**